test/integration/serviceaccount/service_account_test.go

```diff
@@ -285,6 +285,22 @@ func TestServiceAccountTokenAuthentication(t *testing.T) {
 	if err != nil {
 		t.Fatalf("could not delete token: %v", err)
 	}
+	// wait for delete to be observed and reacted to via watch
+	err = wait.PollImmediate(100*time.Millisecond, 30*time.Second, func() (bool, error) {
+		_, err := roClient.CoreV1().Secrets(myns).List(context.TODO(), metav1.ListOptions{})
```

Comment:
why List and not Get?

Reply:
ah, you use a client with that BearerToken, just want to check that is unauhtorized